vhdbuilder/packer/windows-vhd-builder.json

```diff
@@ -49,10 +50,9 @@
                 "createdBy": "aks-vhd-pipeline",
                 "SkipASMAzSecPack": "True"
             },
-            "virtual_network_resource_group_name": "{{user `vnet_resource_group_name`}}",
-            "virtual_network_name": "vnet",
-            "virtual_network_subnet_name": "subnet",
-            "private_virtual_network_with_public_ip": "true"
+            "virtual_network_resource_group_name": "nodesigtest-agent-pool",
```

Comment:
Variable may be more flexible. BTW: this work is interesting, could we have a wiki to note it?

Reply:
Fine. Modify the use of variable and add the wiki.